picture-ts/src/lib/ui.ts

```
@@ -0,0 +1,461 @@
+                        payload.elapsed = formatTime(elapsed);
+                    }
+
+                    // For streaming responses, we don't know the total in advance
+                    // so we just update the value and use an indeterminate progress bar
+                    this.progressBar.update(this.current, payload);
+                }
+                break;
+            case 'simple':
+                let status = `\r${this.title}: ${this.current} tokens`;
+
+                if (this.showTokensPerSecond) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)